src/helpers/glsl/Particles.vert.glsl

```diff
@@ -0,0 +1,11 @@
+uniform float time;
+uniform float radius;
+uniform float size;
+
+void main() {
+  vec3 n = curlNoise(position + time * 0.005);
```

Comment:
the time multiplicator would be a simple user-configurable option that goes a long way